packages/model/src/working-group.ts

```diff
@@ -57,6 +58,7 @@ export type WorkingGroupDataObject = {
   pointOfContact?: WorkingGroupLeader;
   complete: boolean;
   shortText: string;
+  calendars: ReadonlyArray<CalendarResponse>;
```

Comment:
This should be:
```suggestion
  calendars: CalendarDataObject[];
```
You are mixing types from different abstractions. There is a clear line between data providers and controllers. In terms of calendars the data provider returns all data from the database (CMS) and then the controller only picks the stuff that we want to return to the API (as that is it's responsibility as a Controller), see here:
https://github.com/yldio/asap-hub/blob/789ba1b2a27f01d360f5d12a96b936dc93308c38/apps/crn-server/src/controllers/calendars.ts#L58-L67

Similarly - in here you might need to return the entire CalendarDataObject and then parse this in the controller to reduce it but that it the whole point.